site/pages/technology/newsletter-slice/before-sign-up/index.js

```diff
@@ -11,6 +13,9 @@ class BeforeSignUp extends Component {
     this.state = {
       email_address: '', // eslint-disable-line camelcase
       submitting: false,
+      interests: {
+        [groups.techPageSignup]: true,
```

Comment:
There seem to be different values here (`groups.techPageSignup`) and what is defined in the groups (`groups.techNewsletter`). Can I suggest it be called just `tech` since it itself is the group name?